scripts/manage/list_samples.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+bold() {
+  echo ". $(tput bold)" "$*" "$(tput sgr0)";
+}
+
+bold "Here is a list of sample applications to available to install. Selecting one will launch
+a tutorial to install it."
+
+PS3='Please enter your choice: '
+
+tutorials=(
```

Comment:
How difficult would it be to list the directories under `samples` and ask the user to pick one of those? By convention, we could then say that the tutorial should just be called `install.md` in the directory of the sample.

Reply:
Done